client/webpack.config.js

```diff
@@ -1,16 +1,17 @@
 "use strict";
```

Comment:
If you clean this file, can you also fix this with a single quote and no semi-colon? thanks.